azure-pipelines.yml

```diff
@@ -170,17 +170,28 @@ stages:
         displayName: '.Net UTs'
 
       - task: PublishPipelineArtifact@1
+        condition: always()
```

Comment:
Without this, the step wouldn't run because the UT step fails?

Reply:
Exactly. I also tried "ContinueOnError" on the test task in commit b1c554e26199d98cc0868547e390cd552907440c, but if we do that, the step status is "warning" instead of "failed". `condition: always()` seemed preferable therefore.